ui/src/app/components/object-detail/object-detail.component.html

```diff
@@ -90,7 +90,7 @@ <h3 class="title">
           </ng-container>
 
           <ng-container matColumnDef="spDatabase">
```

Comment:
The comment in SQL tab is not aligned : https://drive.google.com/file/d/1oeFzN-GhJx21-QxFfB1fWw32Inb6I5EL/view?usp=sharing&resourcekey=0-XIJHMXNyINo7xaEE8xKsCg